`Paging works` test sometimes fails

Feels like a kind of race condition with a shared resource, didn't look into it yet, just leaving a record

```
1) Setup users list Paging works:

      AssertionError: expected 'dummy-2' to equal 'dummy-1'
      + expected - actual

      -dummy-2
      +dummy-1

      at _callee9$ (tests/users.js:131:33)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:65:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:303:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/regenerator-runtime/runtime.js:117:21)
      at step (tests/users.js:3:191)
      at tests/users.js:3:361
      at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
      at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:510:31)
      at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:567:18)
      at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:612:10)
      at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:691:18)
      at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
      at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
      at Immediate.Async.drainQueues (node_modules/bluebird/js/release/async.js:17:14)
```

--- a/tests/users.js
+++ b/tests/users.js
@@ -32,6 +32,8 @@ run('users', () => {
         gender: 'unknown',
         created_on: moment(new Date()).toISOString()
       }
+
+      await Promise.delay(1)
       await knex('users')
         .insert(userRow)
         .then()

